src/burnfee.rs

```diff
@@ -0,0 +1,106 @@
+pub const HEARTBEAT: u64 = 30_000;
+/// The Burnfee object which contains our starting value and current value when the fee was paid
+#[derive(PartialEq, Clone, Debug)]
+pub struct BurnFee {
+    pub start: f64,
+    pub current: u64,
+}
+
+impl BurnFee {
+    /// Returns the BurnFee used to calculate the work needed to produce a block
+    ///
+    /// * `start` - y-value at x = 0
+    /// * `current` - y-value at x = 0 for next bloc
+    pub fn new(start: f64, current: u64) -> Self {
+        return Self {
```

Comment:
Heard the in the rust community it's more common to use the short return:

```
Self {
  start,
  current
}
```

This can be applied to other `return`s as well.